### DISCUSSION TITLE
WIX Toolset for Dummies

### DISCUSSION THREAD

**Discussion by Papa-26** [wixtoolset/issues#7203] [Questions]
I have ported a large legacy java-8 application into  the 2020s. Creating a binary for Linux was easy. I created a running application for Windows (Intellij with Gradle), but now I have to package it into an msi or exe form. Somebody suggested WIX Toolset, and I am lost.
The documentaions on the WIX-Toolset site doesn't help, since the examples pack trivial HelloWorld type projects. Where can I find instructions for using the toolset for my kind of problem?

**Comment by barnson** [comment]
https://wixtoolset.org/docs/gethelp/ has a bunch of links.

**Reply by Papa-26** [reply]
Thanks, can you recommend the cookbook?

**Reply by barnson** [reply]
Haven't read it.

**Comment by robmen** [comment]
I've been running a weekly live YouTube show for getting started with WiX v4 called "[Deployment Dojo](https://youtube.com/playlist?list=PLDlzbQXIs18slmqmdlS10_de_Cps-QRg6)".

**Comment by Papa-26** [comment]
Thank you Rob,
The problem is that I simply have to pack an .exe, a runtime folder, and an .icon into an.msi.
And in order to learn how to do this monumental task, I have to study two rather boring books - a chore when you are 84.
Ralph

⁣Sent from BlueMail ​

**Reply by robmen** [reply]
It sounds like you just need to read the first five pages of the [tutorial provided by FireGiant](https://www.firegiant.com/wix/tutorial/).

**Comment by ChristianSauer** [comment]
I suggest taking a look at https://github.com/oleg-shilo/wixsharp or maybe https://jrsoftware.org/isinfo.php.
Both are much easier (and better documented) than wix.

**Reply by robmen** [reply]
I found the [WiX Tutorial](https://www.firegiant.com/wix/tutorial/) to be pretty complete. If there are topics that you don't think it covers, I'd suggest [contacting us](https://www.firegiant.com/contact-us/).
